### PR TITLE
updates drag background image size in table

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2015-11-19 at 10 52 47|screen shot 2015-11-19 at 11 15 08|

### CODE REVIEW

**Comment by FinalAngel** [comment]
the background image looks a bit too much to the right, can't that be prevented?

File: djangocms_admin_style/sass/components/_tables.scss [hunk 1]
```diff
@@ -144,8 +144,11 @@
             &.drag {
                 width: 10px !important;
                 padding: 0 !important;
-                background: url('../img/drag.png') !important;
-                background-size: 5px !important;
+                 // adds drag image without background repeat-x to have fixed background image width #252
```
Comment (FinalAngel) [inline djangocms_admin_style/sass/components/_tables.scss]: space

**Comment by lory87** [comment]
@FinalAngel I aligned it with table title.

File: djangocms_admin_style/sass/components/_tables.scss [hunk 2]
```diff
@@ -144,8 +144,11 @@
...
+                background: url('../img/drag-norepeat.png') !important;
+                background-size: 12px 5px !important;
+                background-repeat: repeat-y !important;
```
Comment (FinalAngel) [inline djangocms_admin_style/sass/components/_tables.scss]: you might want to combine this inside ``background:``, same as with position

File: djangocms_admin_style/sass/components/_tables.scss [hunk 3]
```diff
@@ -144,8 +144,11 @@
...
+                background-position-x: 10px !important;
```
Comment (FinalAngel) [inline djangocms_admin_style/sass/components/_tables.scss]: I don't think you need this, that looks a bit off

**Comment by FinalAngel** [comment]
It should look something like this: 
image

**Comment by lory87** [comment]
@FinalAngel updated looks much better now thanks
screen shot 2015-11-19 at 11 38 28

**Comment by FinalAngel** [comment]
great work @lory87 LGTM

**Pull request merged**